modules/ref_ipi-installation-overview.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="installation-overview_{context}"]
+= Installation overview
+
+The installation program supports interactive mode, but Red{nbsp}Hat recommends preparing an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts, and the relevant cluster details.
+
+The installation program loads the `install-config.yaml` file, while the administrator generates the manifests and verifies all prerequisites.
+
+The installation program enrolls all nodes in the cluster and starts the bootstrap VM. The installation program starts the metal platform components as `systemd` services, which have the following containers:
+
+* Ironic-dnsmasq: The DHCP server responsible for handing over the IP addresses to the provisioning interface of various nodes on the provisioning network. Ironic-dnsmasq is only enabled when you deploy an {product-title} cluster with a provisioning network.
+* Ironic-httpd: The HTTP server used to ship the images to the nodes.
+* Image-customization
+* Ironic
+* Ironic-inspector (available in {product-title} 4.16 and earlier)
+* Ironic-ramdisk-logs
+* Extract-machine-os
+* Provisioning-interface: The service that configures networking when the provisioning interface is enabled.
+* Metal3-baremetal-operator: The operator that provides a higher-level API for Ironic and replaces `terraform-provider-ironic` (available in {product-title} 4.16 and later)
+
+The nodes enter the validation phase, where each node moves to a “manageable” state after Ironic validates the credentials to access the Baseboard Management Controller (BMC).
+
+Once in the “manageable” state, the “inspection” phase starts. The "inspection" phase ensures that the hardware meets the minimum requirements needed for a successful deployment of {product-title}.
```

Comment:
I'm not sure the double-quotes are right here.  Are these states reflected in the GUI and should take back ticks?
Per ISG: [Do not use (once) as a conjunction to mean _after_ or _when_](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o).
```suggestion
The nodes enter the validation phase, where each node moves to a _manageable_ state after Ironic validates the credentials to access the Baseboard Management Controller (BMC).

When the node is in the manageable state, the _inspection_ phase starts. The inspection phase ensures that the hardware meets the minimum requirements needed for a successful deployment of {product-title}.
```